process/headerCheck/headerSignatureVerify.go

```diff
@@ -290,25 +294,43 @@ func (hsv *HeaderSigVerifier) VerifySignatureForHash(header data.HeaderHandler,
 	return multiSigVerifier.VerifyAggregatedSig(pubKeysSigners, hash, signature)
 }
 
-// VerifyHeaderWithProof checks if the proof on the header is correct
-func (hsv *HeaderSigVerifier) VerifyHeaderWithProof(header data.HeaderHandler) error {
+// VerifyHeaderWithPrevProof checks if the proof on the header is correct
```

Comment:
update comment as well.. or maybe consider unexporting this method? it is only used internally now